reinforcement_learning/bandits_recsys_movielens_testbed/bandits_movielens_testbed.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p><code>item_pool_size</code>? same for <code>user_id</code></p><p> </p><p>Should we also explain what rewards are given to the item after the clicked item.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/949/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Yes, changed and added this line "No feedback is generated for the items below the clicked item."</p>